source/slang/slang-ir-com-interface.h

```diff
@@ -10,6 +12,6 @@ class DiagnosticSink;
 /// Lower com interface types.
 /// A use of `IRInterfaceType` with `IRComInterfaceDecoration` will be translated into a `IRComPtr` type.
 /// A use of `IRThisType` with a COM interface will also be translated into a `IRComPtr` type.
-void lowerComInterfaces(IRModule* module, DiagnosticSink* sink);
+void lowerComInterfaces(IRModule* module, ArtifactStyle artifactStyle, DiagnosticSink* sink);
```

Comment:
I am not sure if we want to emit ComPtr types differently on kernel code, but if we need that info, It is probably better to just pass in a `TargetReq` instead.